models/articles.model.js

```diff
@@ -73,15 +73,26 @@ const patchVotes = (articleId, inc_votes) => {
         [articleId, inc_votes]
       );
     })
-    .then((result) => result.rows)
-    .catch((error) => {
-      throw error;
-    });
+    .then((result) => result.rows);
+};
+
+const fetchArticlesByTopic = async (topic) => {
+  const topicResult = await db.query(
+    `SELECT * FROM topics WHERE slug = $1;`,
+    [topic]
```

Comment:
^^^ you shouldn't straight away put in a topic ! You need to check that it's valid first in your controller! 

Also, realistically, you shouldn't have 2 db queries in a model - you should only have one :)